astro/src/content/docs/release-notes/index.mdx

```diff
@@ -38,6 +38,21 @@ import IssueResolvedVia from 'src/components/docs/release-notes/ReleaseNoteIssue
 
 Looking for release notes older than 1.23.0? Look in the [release notes archive](/docs/release-notes/archive). Looking to be [notified of new releases?](/docs/operate/roadmap/releases#release-notifications) <span class="not-prose no-underline"><a class="ml-2" href="/docs/releases.xml"><i class="fas fa-xs fa-rss text-orange-700 text-2xl" width="50px" /></a></span>
 
+<ReleaseNoteHeading version='1.53.3' releaseDate='October 22, 2024'/>
+
+### Fixed
+<Issue issue="2764">
+The CSRF token for federated login added in version `1.47.0` is not applied when all identity providers configured for an application use managed domains, and the OAuth authorize request for the application includes an `idp_hint` parameter.
+</Issue>
+
+<Issue issue="2893,2472">
+FusionAuth is not honoring the `rememberDevice` setting set by a user when performing a federated login. This setting tells FusionAuth whether it should establish an SSO session for the user. FusionAuth will look for this value in the following order: 
+1. The selection previously made by the user (stored in a cookie)
```

Comment:
Mention the `fusionauth.remember-device` cookie by name? I think we call it out with the hosted login cookies.

Reply:
> Mention the fusionauth.remember-device cookie by name? I think we call it out with the hosted login cookies.

This would be ok, but if this is an HTTP only cookie, I don't know that we need to mention it. This is an internal implementation and we could change the cookie name if we wanted to, and it should not affect anyone because this is our application. Ideally we would only describe external contracts and parameters to avoid any unnecessary confusion.